src/components/App.js

```diff
@@ -1,13 +1,29 @@
 import React from 'react';
 import './App.css';
 import Map from "../features/map/Map"
+import SignIn from "../features/signin/SignIn.js"
+import { Route, Switch, BrowserRouter } from "react-router-dom";
 
-function App() {
-  return (
+const TITLE = "TrafficFinder"
+
+class App extends React.Component {
+
+  componentDidMount() {
+    document.title = TITLE;
+  }
+
+  render() {
+    return (
       <div className="App">
-        <Map/>
+        <BrowserRouter>
+         <Switch>
+           <Route exact path="/signin" component={SignIn}/>

```

Comment:
Need to hash out default routing before we merge this change. I'd prefer if you kept the selection functionality separate from the login.

Reply:
Not to mention this makes the "/" path blank.

Reply:
sounds good, I'll check out to a new branch from master and merge the changes there